ReactQt/runtime/src/reactitem.cpp

```diff
@@ -393,6 +412,17 @@ double ReactItem::shadowRadius() const {
 
 void ReactItem::setShadowRadius(double shadowRadius) {}
 
+QVector<float> ReactItem::transform() const {
+    return QVector<float>{};
```

Comment:
@ndpmurphy, please return here the same value that was passed to `setTransform()` function.
We can maintain `QVector<float> _transform` member, save value passed in `setTransform` to it and return it from `transform()`